JSQMessagesDemo/DemoMessagesViewController.m

```diff
@@ -62,6 +62,11 @@ - (void)viewDidLoad
         self.collectionView.collectionViewLayout.outgoingAvatarViewSize = CGSizeZero;
     }
     
+    if ([NSUserDefaults typingIndicatorStyleSettingIsAvatar]) {
+        self.collectionView.typingIndicatorAvatarImage = [UIImage imageNamed:@"demo_avatar_cook"];
+        self.collectionView.typingIndicatorMessage = @"Steve is typing…";
```

Comment:
the demo project should be dynamic. this is always going to show the same image/message